qrl_testing/NodeInterface.py

```diff
@@ -0,0 +1,65 @@
+import grpc
+
+from qrl.generated import qrl_pb2, qrl_pb2_grpc
+
+from qrl.core.Transaction import Transaction
+from qrl.core.Wallet import Wallet, AddressBundle
+
+
+class NodeInterface:
+    """
+    Abstraction on top of gRPC so that CLI, tests that need to communicate 
+    with a running node do not reimplement business logic on top of gRPC.
+    """
+    def __init__(self, ip, port=9009, debug=False):
+       """
+       server format: ip:addr
+       """
+       self.debug = debug
+       self.channel = grpc.insecure_channel(":".join([ip, str(port)]))  # 127.0.0.1, 9009 -> "127.0.0.1:9009"
+       self.stub = qrl_pb2_grpc.PublicAPIStub(self.channel)
+
+    def send(self, from_addr:AddressBundle, to_addr:bytes, amount, fee):
+        transferCoinsReq = qrl_pb2.TransferCoinsReq(
+            address_from=from_addr.address,
+            address_to=to_addr,
+            amount=amount,
+            fee=1,
```

Comment:
here is hardcoded.. that could be a problem because if the intention is to use this class in other tests... this could be problematic